modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -388,6 +388,10 @@ function getComments($issueID)
 
     //looping by reference so can edit in place
     foreach ($unformattedComments as &$comment) {
+        $comment["dateAddedFormatted"] = date(
```

Comment:
Googling brought me this: http://stackoverflow.com/questions/3552461/how-to-format-a-javascript-date

And I didn't want to add a dependency and go through discussion-hell.